### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by Eternalgratis**
- When the popup loads, the webapp retrieves data from the selected API and shows the image of the selected item.

### CODE REVIEW

**Comment by danigalindo10** [comment]
### Great Job 👍🏻 
You did a good job, but there are a few changes that we need to make before merging. Please check on the comments. 😄

File: src/modules/modal/populate.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// import getShows from '../get-shows.js';
+
+// const populate = async () => {
+//     await getShows();
+//     renderShows()
+// };
+
+// export default populate;
```
Comment (danigalindo10) [inline src/modules/modal/populate.js]: Please delete this entire file.

File: src/style.css [hunk 2]
```diff
@@ -211,6 +211,19 @@ span {
...
+  position: fixed;
+  background-color: #e0ce00;
+  padding: 10px;
+  z-index: 999999;
+  top: 0;
+  height: 100vh;
+  opacity: 0.9;
+  backdrop-filter: blur(4.5px);
+  display: flex;
+
```
Comment (danigalindo10) [inline src/style.css]: You forgot to close the brackets in this part. Also, we can add more style to the popup, like center the card on the page.

File: src/modules/modal/modalpopup.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    //     const comm = await getComments(dataid).catch(() => []);
+    //     if (comm.length) {
+    //         createComments(comm);
+    //         const commentCount = getCommentsCount();
+    //         comments.textContent = `comments (${commentCount})`;
+    //     } else {
+    //         comments.textContent = 'comments (0)';
+    //     }
+    // };
+    // commentUpdate();
```
Comment (danigalindo10) [inline src/modules/modal/modalpopup.js]: Please delete the code that you are not going to use.

File: src/modules/modal/modalpopup.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    const imageSummary = document.createElement('div');
+    imageSummary.classList.add('image-summary'); 
+    const imagePopUp = document.createElement('img');
+    imagePopUp.classList.add('image-popup');
+    imagePopUp.src = dataimagemedium;
+
+    const dataSummary = document.createElement('div');
+    dataSummary.classList.add('data-summary');
+    dataSummary.innerHTML = datasummary;
+    imageSummary.append(imagePopUp);
```
Comment (danigalindo10) [inline src/modules/modal/modalpopup.js]: # Important 🔴 🔴 🔴 
**Please don't forget that this branch of the pop-up has to print the title, image, and SUMMARY of the series.
In you next branch you must show: the number of comments, the comments, and a functional form to submit the comments and post on the API.**

**Review by danigalindo10 — CHANGES_REQUESTED** [review]
### Important
Please check the individual comments.

**Comment by danigalindo10** [comment]
Please don't forget to fix the linter errors before submitting the changes on the branch. 😄

File: src/style.css (hunk 2 above)
Comment (Eternalgratis) [reply src/style.css]: Thanks for the review. But there's already a closed bracket. The card is also in the center. The element class used to apply the center property is called modal-popup not modal. You can check it

**Pull request merged**